src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,252 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Random;
+import java.io.FileWriter;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.io.File;
+import duke.task.Task;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+public class Duke {
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static final String FILE_SEPARATOR = "CHOPCHOP";
+
+    private static void printSeparator() {
+        for(int i = 0; i < 60; i++) {
```

Comment:
60 seems to be a [magic number](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers) that can be avoided.